src/System.Text.Utf8String/System/Text/Utf8String.cs

```diff
@@ -35,25 +35,6 @@ public Utf8String(byte[] utf8bytes, int index, int length)
             _buffer = new ReadOnlySpan<byte>(utf8bytes, index, length);
         }
 
-        // TODO: reevaluate implementation
-        public Utf8String(IEnumerable<uint> codePoints)
```

Comment:
Yes. The chunk of code that was using it was written slightly different and it allowed me to remove an extra encoder method.